discovery-provider/src/queries/get_underground_trending.py

```diff
@@ -31,48 +31,9 @@
 q = 50
 o = 21
 f = 7
-b = 5
-qw = 50
-hg = 1
-ie = 0.25
-pn = 0.01
-u = 30.0
-qq = 0.001
-oi = 20
-nb = 750
-om = 1500
 qr = 10
```

Comment:
i think it's very likely we'll want to tune these variables too, so maybe we want to incorporate this into the strategy too

Reply:
💯